server/src/main/java/org/opensearch/index/fielddata/FieldDataStats.java

```diff
@@ -80,6 +80,9 @@ public FieldDataStats(long memorySize, long evictions, @Nullable FieldMemoryStat
     }
 
     public void add(FieldDataStats stats) {
+        if (stats == null) {
```

Comment:
When can this be null?

Reply:
It can be null when the client doesn't request fielddata as part of index_metrics.